### DISCUSSION TITLE
ElevenClock opens Multiple Instances on Win-startup

### DISCUSSION THREAD

**Discussion by Yubasha** [martinet101/ElevenClock#1105] [General]
After booting Windows, there are always 2 instances running, which makes it flicker. Closing one solves the Problem, but i was wondering if there is a permanent solution?

**Comment by marticliment** [comment]
Can you please try resetting ElevenClock?

**Reply by Yubasha** [reply]
Well, it did help.

**Reply by marticliment** [reply]
if the issue reappears please repost here
